src/graphql-server/schema.graphql

```diff
@@ -106,6 +111,7 @@ type HposStatus {
 
 type HposAuth {
   isAuthed: Boolean
+  hostSettings: HostSettings
```

Comment:
I meant: call this variable `hposSettings`, so the whole line would be `hposSettings: HposSettings`.

I don't think we need to add a new type here, think we can just reuse `HposSettings`.

Can you rename it, and remove the newly added `HostSettings` type?

Other than that, this PR is good to go. Feel free to merge once this change is made.

Reply:
Got it.  Agree.  Updated.